app/controllers/api/v1/surveys_controller.rb

```diff
@@ -66,8 +66,17 @@ def update
       rescue ActionController::ParameterMissing => error
         render json: {error: error.message}
       end
-    else
-      render json: {notice: 'Failure! Cannot update live survey'}
+    else 
+      #render json: {notice: 'Failure! Cannot update live survey'}
+      begin
+        params = survey_params_update
```

Comment:
This should not go here. Check the flow on how we determine if we a survey can be updated on the notion document here. https://www.notion.so/86c4b6dbae1a483a8c83559d8f6a86ac?v=ccab665d48a74564933708ad994ef8d1&p=e5f69ba4fbea4bdabb27a20ad47fb961

You want to remove the initial islive check on line 54 and then follow the checks in the tech doc